docs/ModelZoo.md

```diff
@@ -0,0 +1,1024 @@
+# NNCF Compressed Model Zoo
```

Comment:
`Model zoo` is it valide name for examples of using? may be better something like `Model Experiments` or 
`Model Results`?

Reply:
I guess it depends on the context. In the main README.md, this table was called **Model Zoo**, because it is common to name so such set of obtained models in other repositories. When these table were in each example's readme they could be referred to as **Results**, because they represented the results of the provided instructions. 

I would keep it as Model Zoo because we now mainly use it as in the first case.